graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/analysis/undirected/ShortestPath.kt

```diff
@@ -19,6 +20,22 @@ class ShortestPath: GraphScenario() {
         }
         then {
             hasShortestPath(graphA, "a", "c", "b")
+            hasShortestPath(graphA, "e", "c")
+        }
+    }
+
+    @AnalysisScenario
+    fun graphWithFewerEdges() {
```

Comment:
❓ I'm not sure I understand the point of this. The name, at least, is not indicative of the intent of the test, nor of what information it adds over the previous one.